packages/docs/.vuepress/config.js

```diff
@@ -18,58 +19,48 @@ module.exports = {
-          { title: 'Meter', link: '/components/meter', isVisible: false },
-          { title: 'Modal', link: '/components/modal', isVisible: true },
-          { title: 'Navigation', link: '/components/navigation', isVisible: false },
-          { title: 'Number Input', link: '/components/number-input', isVisible: false },
-          { title: 'Radio Button', link: '/components/radio-button', isVisible: true },
-          { title: 'Select', link: '/components/select', isVisible: true },
-          { title: 'Status', link: '/components/status/', isVisible: true  },
-          { title: 'Switch', link: '/components/switch/', isVisible: false },
-          { title: 'Tab', link: '/components/tab/', isVisible: true},
-          { title: 'Tag', link: '/components/tag/', isVisible: true },
-          { title: 'Table', link: '/components/table', isVisible: true },
-          { title: 'Text Input', link: '/components/text-input', isVisible: true },
-          { title: 'Toast', link: '/components/toast', isVisible: true },
-          { title: 'Tooltip', link: '/components/tooltip', isVisible: true },
-          { title: 'Top Bar', link: '/components/top-bar', isVisible: false },
-        ],
-      },
-      {
-        children: [
-          { title: 'Changelog', link: 'https://github.com/okta/odyssey/blob/master/packages/odyssey/CHANGELOG.md', isVisible: true},
-        ]
-      },
-    ]
+    flags: {
+      hasSearch: false
+    },
+    nav: {
+      primary: [
+        {
+          title: 'Base',
+          link: '/base/',
+          children: [
+            { title: 'Color', link: '/base/color/' },
+            { title: 'Design Tokens', link: '/base/tokens/' },
+            { title: 'Elements', link: '/base/elements/' },
+            { title: 'Typography', link: '/base/typography/' },
+          ]
+        },
+        {
+          title: 'Components',
+          link: '/components/',
+          children: [
+            { title: 'Button', link: '/components/button/' },
+            { title: 'Checkbox', link: '/components/checkbox/' },
+            { title: 'Link', link: '/components/link/' },
+            { title: 'Modal', link: '/components/modal/' },
+            { title: 'Radio Button', link: '/components/radio-button/' },
+            { title: 'Select', link: '/components/select/' },
+            { title: 'Status', link: '/components/status/' },
+            { title: 'Tab', link: '/components/tab/' },
+            { title: 'Table', link: '/components/table/' },
+            { title: 'Tag', link: '/components/tag/' },
+            { title: 'Text Input', link: '/components/text-input/' },
+            { title: 'Toast', link: '/components/toast/' },
+            { title: 'Tooltip', link: '/components/tooltip/' }
+          ],
+        },
+        { title: 'Icons', link: '/icons/' },
+        { title: 'Figma Kit', link: 'https://www.figma.com/files/676870123940302956/project/2512934/Odyssey-UI' },
```

Comment:
Good with me. Carbon and SFLDS both do this with the private links - external users are met with the sign-in wall.

Since this project is OSS, we may want to supply externally available files at some point *or* add an affordance for indicating when something is private to save people a click.

@joejezowski-okta thoughts here?

Reply:
Moved conversation to new (smaller) PR: https://github.com/okta/odyssey/pull/728/files#r509845137